electrical/breakbeam.py

```diff
@@ -0,0 +1,47 @@
+import RPi.GPIO as GPIO
+import time
+
+class Breakbeam:
+    ACCEPTABLE_TIME = 50 # in seconds
+    BEAM_PINS = {17:'50%_1', 22:'50%_2', 23:'100%_1', 24:'100%_2'}
+
+    def __init__(self):
+        for BEAM_PIN in Breakbeam.BEAM_PINS.keys():
+            GPIO.setmode(GPIO.BCM)
+            GPIO.setup(BEAM_PIN, GPIO.IN, pull_up_down=GPIO.PUD_UP)
+            GPIO.add_event_detect(BEAM_PIN, GPIO.BOTH, callback=self.break_beam_callback)
+            print(self.timer(BEAM_PIN))
```

Comment:
```suggestion
           self.timer(BEAM_PIN)
```
do we need the print statement? event to print timer has its own print statements in the function

Reply:
I don't think we do, dunno why we have that there whoops